libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1336,7 +1336,7 @@ bool TypeChecker::visit(Conditional const& _conditional)
 					_conditional.location(),
 					"True expression's type " +
 					trueType->toString() +
-					" doesn't match false expression's type " +
+					" does not match false expression's type " +
```

Comment:
Actually, I think the ``'s`` aren't that idiomatic either actually... if you want, you could change it further to:
``The type of the true expression ... does not match the type of the false expression ... .``
But both is fine.

Reply:
will do